packages/svelte/types/index.d.ts

```diff
@@ -541,6 +541,8 @@ declare module 'svelte/compiler' {
 			 */
 			runes: boolean;
 		};
+		/** The AST */
+		ast: any;
```

Comment:
any because I couldn't be bothered to type this out as "if modernAst is true, it's this shape, else that shape", and a union of both would make everyone go type cast